client/src/components/PacmanGame/__tests__/gameCore.test.js

```diff
@@ -0,0 +1,78 @@
+import {
+  initSquareGridState,
+  entityToCode,
+  codeToEntity,
+  entityApplier,
+} from '../gameCore';
+
+
+describe('initSquareGridState', () => {
+  const threeZeroGrid = [0, 0, 0];
+  const threeByThreeGrid = [[...threeZeroGrid], [...threeZeroGrid], [...threeZeroGrid]];
+  it('should generates the square grid', () => {
+    expect(initSquareGridState(3)).toEqual(threeByThreeGrid);
+  });
+  it('should generates 2d array with correct number of rows', () => {
```

Comment:
No point of checking the length after comparing the array. I guess it should come before.

Reply:
Yes, this should come before.